tokio/src/io/split.rs

```diff
@@ -24,6 +24,7 @@ cfg_io_util! {
     /// The writable half of a value returned from [`split`](split()).
     pub struct WriteHalf<T> {
         inner: Arc<Inner<T>>,
+        is_write_vectored: bool,
```

Comment:
I'd like to store this in `Inner` instead. It's good for performance for this type to be pointer sized, and due to alignment, this probably doesn't even change the size of `Inner` under most circumstances.

Reply:
Done.